.github/workflows/ci-cd.yml

```diff
@@ -28,105 +25,37 @@ jobs:
-      run: | 
-        VERSION=$GITHUB_REF 
-        VERSION=${VERSION:10:12} NETWORK=qa yarn build:app
-
-    - name: Build in case main
-      if: | 
-        github.event_name == 'release' &&
-        github.event.release.prerelease != true   
-      run: |
-        VERSION=$GITHUB_REF 
-        VERSION=${VERSION:10:12} NETWORK=main yarn build:app
-
-    - name: Copying files to staging server
-      if: ${{ github.ref == 'refs/heads/development' }}
-      uses: appleboy/scp-action@master
-      with:
-        host: ${{ secrets.DEV_REMOTE_HOST }}
-        username: ${{ secrets.DEV_REMOTE_USER }}
-        key: ${{ secrets.DEV_SSH_KEY }}
-        passphrase: ${{ secrets.DEV_SSH_KEY_PASSPHRASE }}
-        rm: true
-        port: ${{ secrets.PORT }}
-        source: "dist/"
-        target: "${{ secrets.DEV_REMOTE_DIR }}"
-
-    - name: Copying files to testing server - Pre-release
-      if: |
-        github.event.release.prerelease == true &&
-        contains(github.event.release.tag_name, 'rc')
-      uses: appleboy/scp-action@master
-      with:
-        host: ${{ secrets.DEV_REMOTE_HOST }}
-        username: ${{ secrets.DEV_REMOTE_USER }}
-        key: ${{ secrets.DEV_SSH_KEY }}
-        passphrase: ${{ secrets.DEV_SSH_KEY_PASSPHRASE }}
-        rm: true
-        port: ${{ secrets.PORT }}
-        source: "dist/"
-        target: "${{ secrets.TEST_REMOTE_DIR }}"
-
-    - name: Copying files to qa server - Pre-release
-      if: |
-        github.event.release.prerelease == true &&
-        contains(github.event.release.tag_name, 'qa')
-      uses: appleboy/scp-action@master
-      with:
-        host: ${{ secrets.DEV_REMOTE_HOST }}
-        username: ${{ secrets.DEV_REMOTE_USER }}
-        key: ${{ secrets.DEV_SSH_KEY }}
-        passphrase: ${{ secrets.DEV_SSH_KEY_PASSPHRASE }}
-        rm: true
-        port: ${{ secrets.PORT }}
-        source: "dist/"
-        target: "${{ secrets.QA_REMOTE_DIR }}"
-
-    - name: Copying files to production server - Release
-      if: |
-        github.event_name == 'release' && 
-        github.event.release.prerelease != true
-      uses: appleboy/scp-action@master
-      with:
-        host: ${{ secrets.PROD_REMOTE_HOST }}
-        username: ${{ secrets.PROD_REMOTE_USER }}
-        key: ${{ secrets.PROD_SSH_KEY }}
-        passphrase: ${{ secrets.PROD_SSH_KEY_PASSPHRASE }}
-        rm: true
-        source: "dist/"
-        target: "${{ secrets.PROD_REMOTE_DIR }}"
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "yarn"
+          cache-dependency-path: "**/yarn.lock"
+
+      - name: Install dependencies
+        run: |
+          yarn install
+          cd playground
+          yarn install
+          cd ..
+
+      - name: Build in case development
+        if: ${{ github.ref == 'refs/heads/development' }}
```

Comment:
no need for this condition anymore, it will only run on the development branch

Reply:
removed.